app/models/User.coffee

```diff
@@ -373,6 +373,14 @@ module.exports = class User extends CocoModel
     # NOTE: Default type is 'course' if no type is marked on the user's copy
     return @get('coursePrepaid')?.type or 'course'
 
+  prepaidFakeType: ->
```

Comment:
I don't think this method name, or duplicating the `prepaidType` logic, is very clear, and it will be hard for future engineers to keep straight what this does or when to use one vs. the other.

How about keeping one `prepaidType` method but adding an argument on whether to include the customized courses?

```coffeescript
user.prepaidType()  # 'course'
user.prepaidType({includeCourseIDs: true})  # 'CS1+CS2+CS3+CS4'
```

Alternatively, we could rename/rewrite this as an `prepaidIncludedCourseDescription` function that returns `'CS1+CS2+CS3+CS4` if it's a customized license, otherwise `null`, so that when we call it, we can use it if it exists and otherwise fall back to calling `prepaidType()`:

```javascript
var prepaidType = view.user.prepaidIncludedCourseDescription() || view.user.prepaidType()
```

Reply:
good point.